scripts/e2e.sh

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env bash
+
+git_ref=$(git rev-parse --short HEAD)
+tower_openshift_project=tower-e2e-"$git_ref"
+
+echo "tower_openshift_project: '"$tower_openshift_project"'" >> e2e_test_config.yml
```

Comment:
`tower_openshift_project ` is lower case but other variables are uppercase? Is this intentional?

Reply:
This is more to do with my nube-ness when it comes to writing bash scripts. Anything that was an env var on the system is upper case. But it seemed to me like the syntax for variables inside a bash script was lower case.

Reply:
Ok, cool. Makes sense. Just wasn't sure if it was intentional or not 👍